.github/workflows/e2e-crossplatform.yml

```diff
@@ -0,0 +1,119 @@
+
+name: E2E cross platform
+
+on:
+  pull_request:
+
+jobs:
+  mac:
+    runs-on: macos-latest
+    timeout-minutes: 30
+    env:
+      TEST_SYSTEM: mac
+      ELECTRON_CUSTOM_VERSION: 19.0.5
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Setup environment"
+        uses: ./.github/actions/setup-env
+        with:
+          bootstrap-packages: "@quiet/logger,@quiet/state-manager,@quiet/backend,@quiet/identity,quiet,backend-bundle,create-require-loader,e2e-tests"
+
+      - name: Download DMG
+        working-directory: ./packages/e2e-tests/Quiet
+        run: curl -LO https://github.com/TryQuiet/quiet/releases/download/quiet%400.19.17-alpha.0/Quiet-0.19.17-alpha.0.dmg
```

Comment:
Could it be dynamically set? I mean e.g. we could have some alias for the latest release? Otherwise we'd have to remember to update it here every time we do a release

Reply:
I think we should do the following:
1. workflows that builds DMG uploads it as artifact using https://github.com/actions/upload-artifact
2. it triggers this workflow
3. this workflow uses https://github.com/actions/download-artifact to download artifact

Reply:
In the current approach, installers are taken from github releases, because that was faster way for me to build these workflows but in the future the application will be built on CI. I will create another task for it and add note to README